src/components/EventListCard/Modal/EventListCardModals.tsx

```
@@ -139,157 +94,40 @@
-            : undefined,
+          input: {
+            id: eventListCardProps._id,
+            name: formState.title,
+            description: formState.eventdescrip,
+            isPublic: publicchecked,
+            isRegisterable: registrablechecked,
+            allDay: alldaychecked,
+            startAt: alldaychecked
+              ? dayjs(eventStartDate).startOf('day').toISOString()
+              : dayjs(eventStartDate)
+                  .hour(parseInt(formState.startTime.split(':')[0]))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)